CHANGELOG.md

```diff
@@ -112,6 +162,57 @@ BUG FIXES:
 * template: Fixed a bug where the `splay` timeout was not being applied when `change_mode` was set to `script`. [[GH-14749](https://github.com/hashicorp/nomad/issues/14749)]
 * ui: Remove extra space when displaying the version in the menu footer. [[GH-14457](https://github.com/hashicorp/nomad/issues/14457)]
 
+## 1.3.7 (October 26, 2022)
```

Comment:
Oh weird, was this auto-generated? It needs to go above the existing [`1.3.6`](https://github.com/hashicorp/nomad/pull/15062/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR131) and the extra one below can be removed.

Reply:
Oh, whoops! No, it wasn't autogenerated. Somehow when I was searching for the right block for 1.3.7 I couldn't find 1.3.6 and thought it was missing from the previous release. Will fix.

Reply:
Oh I see what happened. It was missing from release/1.4.x so I'd fixed it there, but then when I copied that fix over to main it wasn't necessary. Weird.